onnxruntime/core/framework/session_state.cc

```diff
@@ -190,16 +190,24 @@ static int64_t CalculateMemoryPatternsKey(const std::vector<std::reference_wrapp
 
 #ifdef ENABLE_TRAINING
 namespace {
-Status ResolveDimParams(const GraphViewer& graph, const std::map<std::string, TensorShape>& feeds, std::unordered_map<std::string, int64_t>& out) {
+Status ResolveDimParams(const GraphViewer& graph,
+                        const std::map<std::string, TensorShape>& feeds,
+                        std::unordered_map<std::string, int64_t>& out) {
   for (const auto* input : graph.GetInputs()) {
     auto* shape = input->Shape();
     auto it = feeds.find(input->Name());
     if (it == feeds.end())
-      return Status(ONNXRUNTIME, FAIL, "Graph input " + input->Name() + " is not found in the feed list, unable to resolve the value for dynamic shape.");
+      return Status(ONNXRUNTIME, FAIL,
```

Comment:
> return Status(ONNXRUNTIME, FAIL, [](start = 5, length = 33)

pre-existing: even though it is a single line, consider wrap the if statement with bracet for consistancy.

Reply:
ok.

---
In reply to: [439076901](https://github.com/microsoft/onnxruntime/pull/4204#discussion_r439076901) [](ancestors = 439076901)